rpc/api/net.go

```diff
@@ -7,15 +7,24 @@ import (
 	"github.com/qlcchain/go-qlc/ledger"
 	"github.com/qlcchain/go-qlc/log"
 	"go.uber.org/zap"
+	"time"
 )
 
+const syncTimeout = 10 * time.Second
+
+var lastSyncTime int64
+
 type NetApi struct {
 	ledger *ledger.Ledger
 	eb     event.EventBus
 	logger *zap.SugaredLogger
 }
 
+func syncingTime(t time.Time) {
+	lastSyncTime = t.Add(syncTimeout).UTC().Unix()
```

Comment:
it is not  thread safe, and `eb.Subscribe` is running as goroutines...

Reply:
add lock to ensure thread safe